### ISSUE TITLE
The step debuging features doesn't work on a breakpoint.

### ISSUE THREAD

**Issue by johnfound** [eteran/edb-debugger#756]
After stopping on a break-point, the features for step debugging stop to work. I mean Step-Into, Step-Over. The cursor flashes as it tries to make something, but the instruction is not executed. I have a feeling that it executes the breakpoint again and stops on it, instead of executing the instruction.

EDB version 1.1.0, Compiled from source by the  Arch Linux package manager (AUR repository).

**Comment by eteran** [comment]
That's strange, they obviously should work, but I'll see if I can replicate the issue.

**Comment by johnfound** [comment]
Here are some more details:

Tested the problem with different programs (32bit, 64bit, compiled against MUSL, etc.) and the problem persists.

In addition, if the breakpoint is placed on the "call" instruction, and Step-over is pressed (executed from the menu) additional breakpoint bullet is placed permanently on the next instruction:

https://imgur.com/3t00lj0.png

It can be removed only by "toggle breakpoint" (F2).

**Comment by eteran** [comment]
oh wow, that's pretty bad! I have been able to replicate locally :'-(

**Comment by eteran** [comment]
OK, I'm pretty disappointed with myself here. I created this bug with d5d180a75a6248c9c9e4c2a604793fa4dac8c3ee . It was a stupid issue and will be fixed in master shortly. I'll probably want to do a release ASAP since it made it into 1.1.0 without me noticing :-(.

**Comment by eteran** [comment]
For the curious, it was this erronious change :-(

```
if (!(flags & ResumeFlag::Forced)) {
```

Became

```
if (flags == ResumeFlag::Forced) {
```

in a refactor... which is 100% backwards.

**Comment by eteran** [comment]
Fixed in master/1.2.0. Thanks for the report!
